rails-commenting.rb

```diff
@@ -6,25 +6,25 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) Defines the class of BlogPostsController which Inherits from ApplicationController
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2) Creates an instance variable called @posts that will show all BlogPost(s)
     @posts = BlogPost.all
   end
 
-  # ---3)
+  # ---3) Creates the Show method that will call a single blog post / single piece of information
   def show
     @post = BlogPost.find(params[:id])
   end
 
-  # ---4)
+  # ---4) Creates the New method  that will render a new blog post entry 
```

Comment:
Will it render a new entry or will it render something different?  Take a closer look at the [RESTful routes](https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/rails/restful-routes-crud.md) section in the syllabus.